nodes/Hedera/services/token/TokenService.ts

```diff
@@ -162,6 +236,12 @@ export class TokenService implements IHederaService {
 				params.initialSupply = getNodeParameter('initialSupply', itemIndex);
 				// Add treasury account ID for token creation
 				params.treasuryAccountId = accountId;
+				params.enableSupplyKey = getNodeParameter('enableSupplyKey', itemIndex);
+				break;
+			case 'mint':
+				params.tokenId = getNodeParameter('tokenId', itemIndex);
+				params.supplyKey = getNodeParameter('supplyKey', itemIndex);
+				params.mintAmount = getNodeParameter('mintAmount', itemIndex);
```

Comment:
```suggestion
				params.amount = getNodeParameter('amount', itemIndex);
```

Just `amount` here is sufficient for variable name (will obvs need to update across the board)